exec/java-exec/src/main/java/org/apache/drill/exec/ops/QueryContext.java

```diff
@@ -92,6 +93,7 @@ public QueryContext(final UserSession session, final DrillbitContext drillbitCon
     this.drillbitContext = drillbitContext;
     this.session = session;
     this.queryId = queryId;
+    skipWritingProfile(false);
```

Comment:
```suggestion
    this.skipProfileWrite = false;
```

Reply:
no need to explicitly set skipProfileWrite to false in constructor.